### ISSUE TITLE
[$10][FAST-15] Topcoder Handle contrast between foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3106]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing
2. Check the topcoder handle

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA if you change background color to #181818 New contrast ratio: 4.51
- Passes AA if you change text color to #EB1313 New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 3.93 (foreground color: #ef3a3a, background color: #ffffff, font size: 11.3pt, font weight: normal).

**Device**: Desktop/Labtop
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AAA
**WCAG Success Criterion**: 1.4.6 Contrast (Enhanced)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=0bf9d0fe%2Dbdaa%2D4af3%2Dad48%2Dd3fa9f0e20e1&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2Fcodejam%2F4

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097686 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097686 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@kkartunov I guess even this is already covered in one of PR

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097686 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@kkartunov PR https://github.com/topcoder-platform/community-app/pull/3131

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097686<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
@kkartunov , To test exact scenario, we need different test account with level (eg. black, yellow, red etc). Can you please setup below test account accordingly. 
 Test Accounts: pshah_manager/topcoder123 | pshah_copilot/topcoder123 | lakshmiathreya  >> Topcoder123 

![image](https://user-images.githubusercontent.com/25297859/62111280-f337b700-b2cf-11e9-8fb9-1a5b3c7ba275.png)
